ninja_cursor_pagination/__init__.py

```diff
@@ -1,28 +1,20 @@
-from base64 import b64decode, b64encode
-from dataclasses import dataclass
-from typing import Any
+from collections.abc import Sequence
+from typing import Any, TypedDict, cast
 from urllib import parse
 
-from django.db.models import QuerySet
+from django.db.models import Model, QuerySet
 from django.http import HttpRequest
 from django.utils.translation import gettext as _
 from ninja import Field, Schema
 from ninja.pagination import PaginationBase
 from pydantic import field_validator
 
+from ._cursor import Cursor
 
-@dataclass
-class Cursor:
-    offset: int = 0
-    reverse: bool = False
-    position: str | None = None
+MAX_PAGE_SIZE = 100
```

Comment:
Made this a module-level constant, instead of a class attribute, for scoping reasons.

Reply:
It'd be exactly the same, and they'd have to set it globally for the application either way. 

```python
from ninja_cursor_pagination import CursorPagination

CursorPagination.max_page_size = 200
```
vs
```python
import ninja_cursor_pagination

ninja_cursor_pagination.MAX_PAGE_SIZE = 200
```

Reply:
Ah, I see your point.

If you think this ought to be configurable, then I think we need to make an optional instance argument (and setting it will no longer be global).

Reply:
I do think it should be configurable without too much contortion. In the previous incarnation, that looked like this:

```python
class BiggerPagesPagination(CursorPagination):
    max_page_size = 200
```

If that worked, I think that's good enough. Otherwise an optional argument is fine.